shell/platform/android/io/flutter/plugin/platform/PlatformViewsController.java

```diff
@@ -476,6 +482,12 @@ public void attachToView(@NonNull View flutterView) {
     for (VirtualDisplayController controller : vdControllers.values()) {
       controller.onFlutterViewAttached(flutterView);
     }
+
+    for (int i = 0; i < platformViewParent.size(); i++) {
+      int viewId = platformViewParent.keyAt(i);
+      ((FlutterView) flutterView).addView(platformViewParent.get(viewId));
+      platformViews.get(viewId).onFlutterViewAttached(flutterView);
+    }
```

Comment:
This can be removed if  onFlutterViewAttached is moved to `initializePlatformViewIfNeeded`. `initializePlatformViewIfNeeded` is called when `FlutterView` isn't `null`. Although, I would appreciate a `null` check there!